common/browser_options_test.go

```diff
@@ -31,6 +32,24 @@ func TestBrowserLaunchOptionsParse(t *testing.T) {
 				}, lo)
 			},
 		},
+		"defaults_on_cloud": {
+			onCloud: true,
+			opts: map[string]any{
+				"headless":       false,
+				"devtools":       true,
+				"executablePath": "something else",
+			},
+			assert: func(tb testing.TB, lo *LaunchOptions) {
+				tb.Helper()
+				assert.Equal(t, &LaunchOptions{
+					Env:               make(map[string]string),
+					Headless:          true,
+					LogCategoryFilter: ".*",
+					Timeout:           DefaultTimeout,
+					onCloud:           true,
```

Comment:
Do we need to assert only on `headless`, `devtools` and `executablePath`?

Reply:
Only those fields are forced, and we check whether users can overwrite them in the cloud. The other fields are taken care of already by the other test cases.

Reply:
👍 Resolved in 78c3f7a.